plugin/pkg/auth/authorizer/node/node_authorizer.go

```diff
@@ -98,17 +102,42 @@ func (r *NodeAuthorizer) Authorize(attrs authorizer.Attributes) (authorizer.Deci
 	return authorizer.DecisionNoOpinion, "", nil
 }
 
+// authorizeStatusUpdate authorizes get/update/patch requests to status subresources of the specified type if they are related to the specified node
+func (r *NodeAuthorizer) authorizeStatusUpdate(nodeName string, startingType vertexType, attrs authorizer.Attributes) (authorizer.Decision, string, error) {
+	switch attrs.GetVerb() {
+	case "get", "update", "patch":
+		// ok
+	default:
+		glog.V(2).Infof("NODE DENY: %s %#v", nodeName, attrs)
+		return authorizer.DecisionNoOpinion, "can only get/update/patch this type", nil
+	}
+
+	if attrs.GetSubresource() != "status" {
+		glog.V(2).Infof("NODE DENY: %s %#v", nodeName, attrs)
+		return authorizer.DecisionNoOpinion, "cannot get subresource", nil
+	}
+
+	return r.authorize(nodeName, startingType, attrs)
+}
+
 // authorizeGet authorizes "get" requests to objects of the specified type if they are related to the specified node
 func (r *NodeAuthorizer) authorizeGet(nodeName string, startingType vertexType, attrs authorizer.Attributes) (authorizer.Decision, string, error) {
-	if attrs.GetVerb() != "get" || len(attrs.GetName()) == 0 {
+	if attrs.GetVerb() != "get" {
```

Comment:
why this change?

Reply:
I am not entirely sure (it was there in your original PR).  As I understand , name will not be available in all requests but ideally should always be available for Get requests. I don't know what the Name check guards against.

Reply:
Name check is still present in `authorize` function below.

Reply:
ah, that's why :)